### PR TITLE
Player Commands

### PR DESCRIPTION
**Pull request by Fortifier42**

### CODE REVIEW

File: dEssentials/dEssentials-Player.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+dessentials_command_heal:
+  type: command
+  debug: false
+  name: heal
+  description: Heals yourself or another player to full health.
+  usage: /heal [player]
+  permission: denizen.essentials.admin.heal
+  tab complete:
+    - determine <server.list_online_players.parse[name].filter[starts_with[<context.args.last>]]>
```
Comment (mcmonkey4eva) [inline dEssentials/dEssentials-Player.yml]: Isn't this like... an unaltered default?

File: dEssentials/dEssentials-Player.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - define player <player>
+    - if <context.args.size> == 1 {
+      - define player <server.match_player[<context.args.get[1]>]||null>
+    }
+    - if <def[player]> == null {
+      - narrate "<&c>Invalid player!"
+      - queue clear
+    }
+    - heal <def[player]>
+    - narrate "<&2>You healed <&b><t[<def[Player].is[==].to[<player>]>]:Yourself||<def[Player].name>><&a>."
```
Comment (mcmonkey4eva) [inline dEssentials/dEssentials-Player.yml]: ffs just use an if

File: dEssentials/dEssentials-Player.yml (hunk 1 above)
Comment (Fortifier42) [inline dEssentials/dEssentials-Player.yml]: There isn't a default /heal command?

Comment (mcmonkey4eva) [inline dEssentials/dEssentials-Player.yml]: ... tab complete section, foo.

**Pull request closed without merging**